workflow_examples/jenkins/rapidastscan.groovy

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env groovy
+
+
+podTemplate(
+        containers: [
+            containerTemplate(
+                name: 'rapidast',
+                image: "quay.io/insights-qe/rapidast:multiuser",
```

Comment:
this will need to point out to the https://quay.io/redhatproductsecurity/rapidast

Reply:
Code in the pipeline relies on the git to be present in the image. I can update this, but the pipeline will break with the current release of the  multiuser image, we need a new build with latest dev

Reply:
I see. I'll build a new one soon.

Reply:
@san7ket quay.io/repository/redhatproductsecurity/rapidast:2.2.1 is now available. pls check.

Reply:
updated